system/ee/ExpressionEngine/Addons/member/mod.member.php

```diff
@@ -2619,6 +2619,42 @@ public function custom_profile_data($typography = true)
         return ee()->TMPL->tagdata;
     }
 
+    /**
+     * Member Role Groups list
+     */
+    public function role_groups()
+    {
+        $member_id = (!ee()->TMPL->fetch_param('member_id')) ? ee()->session->userdata('member_id') : ee()->TMPL->fetch_param('member_id');
+
+        $member = ee('Model')
+            ->get('Member', $member_id)
+            ->with('RoleGroups')
+            ->all()
+            ->first();
+
+        if (!$member) {
+            return ee()->TMPL->no_results();
+        }
+
+        $vars = [];
+        foreach($member->RoleGroups->asArray() as $roleGroup) {
```

Comment:
Is there a specific reason to call `asArray` here? `RoleGroups` is already iterable

Reply:
Nope, I didn't realize it was iterable 👍